src/cljs/nr/translations.cljs

```diff
@@ -160,6 +160,7 @@
      :max "maximum"
      :influence "Influence"
      :agenda-points "Agenda points"
+     :deck-points "Deck points"
```

Comment:
Make sure to add an entry to the pig latin translation file so you can verify it works.